lib/client.js

```diff
@@ -43,6 +43,50 @@ Freshdesk.prototype = Object.create({
 	 * @property {string} [updated_since]   Updated since.
 	 */
 
+
+  /**
+	 * Create a GET request to the freshDesk api
+   * @param {String} url - Url path excluding base path to the endpoint
+   * @param {Object} query - Query params to be sent
+   * @param callback
+   */
+	getRequest(url, query, callback) {
```

Comment:
this is **NO-NO-NO**. I don't want to add such functions to public interface of this SDK